statsmodels/tsa/stattools.py

```diff
@@ -1055,6 +1055,114 @@ def arma_order_select_ic(y, max_ar=4, max_ma=2, ic='bic', trend='c',
     return Bunch(**res)
 
 
+def kpss(x, null_hypo="level", lags=None):
+    """
+    Kwiatkowski-Phillips-Schmidt-Shin test for stationarity.
+
+    Computes the Kwiatkowski-Phillips-Schmidt-Shin (KPSS) test for the null
+    hypothesis that x is level or trend stationary.
+
+    Parameters
+    ----------
+    x : array_like, 1d
+        Data series
+    null_hypo : str{"level", "trend"}
+        Indicates the null hypothesis for the KPSS test
+        * "level" : The data is level stationary (default)
+        * "trend" : The data is trend stationary
+    lags : int
+        Indicates the number of lags to be used. If None (default),
+        lags is set to int(12 * (n / 100) ** (1 / 4)), as outlined in
+        Schwert (1989).
+
+    Returns
+    -------
+    kpss_stat : float
+        The KPSS test statistic
+    p_value : float
+        The p-value of the test
+    lags : int
+        The truncation lag parameter
+    flag: int
+        * -1 : the p-value is smaller than the indicated p-value.
+        * 0 : the p-value is the indicated p-value.
+        * 1 : the p-value is greater than the indicated p-value.
```

Comment:
This return value wasn't immediately obvious to me until I read the code, but this may just be my lack of familiarity. Do other hypothesis tests in Statsmodels use this?